subsys/logging/log_core.c

```diff
@@ -17,6 +17,10 @@
 #define CONFIG_LOG_PRINTK_MAX_STRING_LENGTH 1
 #endif
 
+#ifdef CONFIG_LOG_FRONTEND
```

Comment:
you don't need to wrap it with #ifdef. Otherwise compiler gives warning when frontend is disabled.